rpc/gnmi/gnmi-specification.md

```diff
@@ -1749,6 +1738,11 @@ limitations under the License
 
 # 7 Revision History
 
+* v0.7.0: April 28, 2022
```

Comment:
I think this is really 0.8.0? I think we have a skew between the proto version and the doc version.

Reply:
You are correct.  We apparently missed the update to 0.7.0 in 2018.  I've added the change history.